docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -1,283 +1,71 @@
 ---
-title: Jobs | Dagster
+title: Jobs
```

Comment:
> > make `concepts/jobs` into a concept/category page

i don't see the `concepts/jobs` in the changes. is this the page you referred as "concepts/jobs"? where do you envision it to live going forward?

Reply:
Yes - sorry that wasn't clear.

Reply:
Yes - sorry that wasn't clear. I'm trying to figure out where to put this in the sidenav. Here's how things are currently structured:

- `Concepts > Assets > Asset jobs`
- `Concepts > Advanced > Ops > Op jobs`

Normally I'd say `Concepts > Jobs`, but we removed that to de-emphasize the concept. We could do this, maybe?:

```
- Concepts
  - Advanced
    - Jobs (/concepts/jobs)
      - Asset jobs (/concepts/jobs/asset-jobs)
      - Op jobs (/concepts/jobs/op-jobs)
```

Reply:
I wonder if it'd make sense to have this for now:
```
- Concepts
  - Assets
        - Asset jobs (/concepts/assets/asset-jobs)
  - Advanced
    - Jobs (/concepts/jobs)
      - Asset jobs (/concepts/assets/asset-jobs)
      - Op jobs (/concepts/jobs/op-jobs)
```
which makes this solution a stopgap while we will be improving the core api.

and then, later this year, once we get rid of asset jobs on the critical path for assets, we can switch to:
```
- Concepts
  - Assets
  - Advanced
    - Jobs (/concepts/jobs)
      - Asset jobs (/concepts/jobs/asset-jobs) # move it down to an advanced concept
      - Op jobs (/concepts/jobs/op-jobs)
```

Reply:
I think that's fine - it's kinda weird but I'd be surprised if anyone was confused by it.

Reply:
I also moved `Job execution` and `Metadata & tags` under this heading, as it felt weird to leave them hanging around below it. So now it's:

```
- Concepts
  - Assets
      - Asset jobs (/concepts/assets/asset-jobs)
  - Advanced
    - Jobs (/concepts/jobs)
      - Asset jobs (/concepts/assets/asset-jobs) # links to the essential page
      - Op jobs (/concepts/jobs/op-jobs)
      - Job execution
      - Job metadata & tags
```

Reply:
this looks good to me! waiting on the deployment and do one last pass before stamping